src/maps/southern_us/settings.ts

```diff
@@ -0,0 +1,35 @@
+import { GameKey } from "../../api/game_key";
+import {
+  EMIL,
+  MapSettings,
+  ReleaseStage,
+} from "../../engine/game/map_settings";
+import { map } from "./grid";
+import {
+  SouthernUSMoveValidator,
+  SouthernUSMoveHelper,
+  SouthernUSMoveAction,
+} from "./move";
+import { SouthernUSGoodsGrowthPhase } from "./goods_growth";
+import { SouthernUSIncomeReductionPhase } from "./income";
+
+export class SouthernUsMapSettings implements MapSettings {
+  readonly key = GameKey.SOUTHERN_US;
+  readonly name = "Southern US";
+  readonly designer = "John Bohrer";
+  readonly implementerId = EMIL;
+  readonly minPlayers = 3;
+  readonly maxPlayers = 6;
+  readonly startingGrid = map;
+  readonly stage = ReleaseStage.DEVELOPMENT;
```

Comment:
The DEVELOPMENT stage hides this map from the drop-down. Would be good to up it to alpha before merging. :)